include/sys/trace_acl.h

```diff
@@ -56,15 +57,15 @@ DECLARE_EVENT_CLASS(zfs_ace_class,
 	    __field(uint64_t,		z_mapcnt)
 	    __field(uint64_t,		z_size)
 	    __field(uint64_t,		z_pflags)
-	    __field(uint64_t,		z_uid)
-	    __field(uint64_t,		z_gid)
 	    __field(uint32_t,		z_sync_cnt)
 	    __field(mode_t,		z_mode)
 	    __field(boolean_t,		z_is_sa)
 	    __field(boolean_t,		z_is_mapped)
 	    __field(boolean_t,		z_is_ctldir)
 	    __field(boolean_t,		z_is_stale)
 
+	    __field(uint16_t,		i_uid)
+	    __field(uint16_t,		i_gid)
```

Comment:
As a matter of fact they should. The reason why I put them to uint16 is because the overflowuid/gid sysctls are set to 65535. Would you care to fix this in the merge or should I send an updated version ?